_sass/docs.scss

```diff
@@ -129,7 +129,7 @@
     &[open] {
 
       summary {
-        background: rgb(114,114,137, 0.07);
+        background: rgba(114,114,137, 0.07);
```

Comment:
@ikbensiep jekyll threw errors on these because there's 4 arguments. Hope this is still correct!

Reply:
Ah, guess I missed this because at one point this pasted from devtools in a weird notation I didn't know about `rgb(114 114 137 / 7%)` which sass also didn't like. Looks good this way!